app/talk/search.cjsx

```diff
@@ -69,6 +69,10 @@ module.exports = React.createClass
     numberOfResults = @state.results.length
 
     <div className="talk-search">
+      <button  className="link-style" type="button" onClick={-> window.history.back()}>
```

Comment:
Could this use https://github.com/rackt/react-router/blob/0.13.x/doc/06%20Mixins/Navigation.md#goback instead?